content/en/docs/concepts/policy/limit-range.md

```diff
@@ -0,0 +1,383 @@
+---
+reviewers:
+- sftim

```

Comment:
Although I'm trying to be helpful with comments, I'm aware that I (still) haven't got my status in the Kubernetes project sorted. That means I'm not officially a reviewer yet.

Reply:
I think, typically, @nelvadas, you would put your name in the file, and add any others that are knowledgeable about the topic. I think that when future changes are made, you would be notified about the changes and could review.
@bradtopol, 1) would you know which SIG this topic falls under and 2) could we ping the group in a Slack channel and ask if there is interest to review?
I read through the manifests and the commands, but I did not execute the commands. I trust that the `math` in the examples is correct -- nothing stood out as a problem to me; I think I could follow the instructions. I still have a few more spelling and word choice nits to comment on, though. Overall, the page is looking much better.